### PR TITLE
PR for watch later page

### PR DESCRIPTION
**Pull request by narharikale**
feat added : Save to watch later

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Good file structure 
Functionality working as expected

File: src/frontend/Router/Router.js [hunk 1]
```diff
@@ -16,6 +18,23 @@ function Routers(){
                     <Route path="/signup" element={<Signup/>}/>
                 </>
             )}
+            <Route path="/watchlater" element={ 
+                <PrivateRoute>
```
Comment (t007rushi) [inline src/frontend/Router/Router.js]: Suggestion : change it to ```RequireAuth``` as per convention

File: src/frontend/screen/Signin.jsx [hunk 2]
```diff
@@ -20,12 +20,12 @@ import { axiosService } from "../services/axiosService";
     let from = location.state?.from?.pathname || "/";
 
     const signinHandler = async( email , password ) => {
-        const { data } = await axiosService('post' , '/api/auth/login', {
+        const response  = await authService('post' , '/api/auth/login', {
```
Comment (t007rushi) [inline src/frontend/screen/Signin.jsx]: destructure here itself

File: src/frontend/services/combinedServices.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import axios from 'axios' ; 
+
+const combinedService = async(method , url , token , video ) => {
```
Comment (t007rushi) [inline src/frontend/services/combinedServices.js:3]: Nicely implemented

**Review by t007rushi — APPROVED** [review]
Look for suugestion 
overall good work 
naming and folder structure

File: src/frontend/screen/Signin.jsx (hunk 2 above)
Comment (narharikale) [reply src/frontend/screen/Signin.jsx]: done!

File: src/frontend/Router/Router.js (hunk 1 above)
Comment (narharikale) [reply src/frontend/Router/Router.js]: changed

**Pull request merged**